app/components/post-title.js

```diff
@@ -1,15 +1,52 @@
 import Ember from 'ember';
 
+/**
+  @class post-title
+  @module Component
+  @extends Ember.Component
+ */
 export default Ember.Component.extend({
   classNames: ['post-title'],
   classNameBindings: ['isEditing:editing'],
 
+  /**
+    @property currentUser
+    @type Ember.Service
+   */
   currentUser: Ember.inject.service(),
 
+  /**
+    Computed alias for `currentUserIsPostAuthor` that returns whether or not the current
+    user can edit the current post.
```

Comment:
I'm not sure if we should describe what it's doing but rather what it's behavior is. "Returns whether or not the current user can edit the current post" seems simpler and better.